egs/wsj/s5/steps/nnet3/libs/rnn_train_lib.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python
+
+
+# Copyright 2016 Vijayaditya Peddinti.
+#           2016 Vimal Manohar
+# Apache 2.0.
+
+import logging
+import imp
+
+imp.load_source('nnet3_train_lib', 'steps/nnet3/nnet3_train_lib.py')
```

Comment:
It looks very redundant. If you just want to import the functions then directly use `from <module> import *`, you would just have to modify the `sys.path`in the script to add the module address as these are not in the syspath by default. regarding the import commands just pick one and stay consistent. BTW I would recommend using the `import <module>` as the namespace specification when calling function makes it easy to debug, and helps avoid issues when functions with similar signatures are defined in different modules (e.g. this happens a lot in python modules for chain and xent training).